isis/src/control/objs/BundleSolutionInfo/BundleSolutionInfo.cpp

```diff
@@ -1374,6 +1481,16 @@ namespace Isis {
       fpOut << (const char*)pointSummaryString.toLatin1().data();
     }
 
+    int nLidarPoints = m_statisticsResults->bundleLidarControlPoints().size();
+    for (int i = 0; i < nLidarPoints; i++) {
+      BundleLidarControlPointQsp lidarControlPoint
+          = m_statisticsResults->bundleLidarControlPoints().at(i);
+
+      QString pointSummaryString =
+          lidarControlPoint->formatBundleOutputSummaryString(berrorProp);
+      fpOut << (const char*)pointSummaryString.toLatin1().data();
+    }
+
```

Comment:
Should this section get a title (Similar to POINTS DETAIL BELOW)? Alternatively, should it be included under POINTS DETAIL below?